pkg/v12/resource/chartoperator/create.go

```diff
@@ -80,18 +78,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 				}
 			}()
 
-			clusterDNSIP, err := key.DNSIP(r.clusterIPRange)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-			v := &Values{
-				ClusterDNSIP: clusterDNSIP,
-				Image: Image{
-					Registry: r.registryDomain,
-				},
-				TillerNamespace: chartOperatorNamespace,
-			}
-			b, err := json.Marshal(v)
+			b, err := json.Marshal(createState.ChartValues)
```

Comment:
This was weird before. We now generate the chart values as part of the desired state.